src/zui/palette/palette.mdx

```diff
@@ -0,0 +1,580 @@
+import { Meta, ColorPalette, ColorItem } from '@storybook/blocks';
+
+<Meta title="Colors" />
+
+# Palette
+
+<ColorPalette>
+  <ColorItem
+    title="Basic"
+    colors={{
+      black: '#000000',
+      white: '#FFFFFF',
```

Comment:
All of the color values in this file are hardcoded, and I don't think they should be. I think the work you have done here to document the colors should be moved over to the branch where you have implemented all the colors in the theme, and the values of the colors here should reference the constants in the theme rather than hardcoded.